scene/2d/audio_stream_player_2d.cpp

```diff
@@ -391,6 +399,18 @@ int AudioStreamPlayer2D::get_max_polyphony() const {
 	return max_polyphony;
 }
 
+void AudioStreamPlayer2D::set_panning_strength(float p_panning_strength) {
+	if (p_panning_strength < 0) {
+		panning_strength = 0;
+	} else {
+		panning_strength = p_panning_strength;
+	}
```

Comment:
If negative values are invalid, I would error out instead of clamping silently:
```suggestion
	ERR_FAIL_COND_MSG(p_panning_strength < 0, "Panning strength must be a positive number.");
	panning_strength = p_panning_strength;
```
Same in 3D.